ompi/mpi/man/man3/MPI_Type_indexed.3in

```diff
@@ -165,6 +165,6 @@ called. By default, this error handler aborts the MPI job, except for I/O functi
 .SH SEE ALSO
 .ft R
 .sp
-MPI_Type_hindexed
+MPI_Type_create_hindexed
```

Comment:
This man page should be removed, just like all the other removed man pages.

Ditto for any other remaining man pages for removed symbols.

Reply:
`MPI_Type_indexed()` (like `MPI_Type_vector()` or `MPI_Type_contigous()`) was never removed nor deprecated in favor of `MPI_Type_create_indexed()`. If you know the rationale, I'll be glad to hear it :-)

Reply:
Right.  Duh!  Disregard my comment.

indexed = int indexing, where hindexed = byte indexing.

Reply:
There was a similar question few days ago on the [MPI Forum github](https://github.com/mpi-forum/mpi-standard/pull/109).

Reply:
@bosilca I am afraid this is a private repository since I cannot access it :-(

Reply:
To summarize in "MPI-2 some of the datatype creation functions were deprecated because there was a mismatch between their C and Fortran interfaces (C API taking MPI_Aint and Fortran API  an int). We deliberately made the decision not to deprecate the other datatype creation function simply to accommodate for a name change."